### PR TITLE
[eos] Implement B -> gamma ell nu signal PDF

### PR DESCRIPTION
**Pull request by mreboud**
I am not sure about the definition of the cos theta_l variable. @pluegh Can you check it?

### CODE REVIEW

File: eos/signal-pdf.cc [hunk 1]
```diff
@@ -248,6 +249,19 @@ namespace eos
                     )
                 ),
 
+            make_signal_pdf("B->gammalnu::d^2Gamma/dEgamma/dcos(theta_l)",
+                    Options{ },
+                    &BToGammaLeptonNeutrino::fully_differential_decay_width,
+                    std::make_tuple(
+                        KinematicRange{ "Egamma", 0.1, 2.64, BToGammaLeptonNeutrino::kinematics_description_Egamma },
```
Comment (mreboud) [inline eos/signal-pdf.cc:256]: This may require a comment...

**Review by pluegh — APPROVED** [review]
Looks good to me except for variable names.
I can fix that in the future, if you want to merge this quickly.

**Review by dvandyk — APPROVED** [review]

File: eos/signal-pdf.cc (hunk 1 above)
Comment (mreboud) [reply eos/signal-pdf.cc:256]: I added that to the description of the kinematic variable `Egamma`.

File: eos/b-decays/b-to-gamma-l-nu.hh [hunk 2]
```diff
@@ -43,6 +43,13 @@ namespace eos
             double differential_decay_width_dEgamma(const double & E_gamma) const;
             double integrated_decay_width(const double & E_gamma_min) const;
 
+            /*!
+             * Descriptions of the process and its kinematics.
+             */
+            static const std::string description;
+            static const std::string kinematics_description_Egamma;
+            static const std::string kinematics_description_c_theta_l;
```
[the remark this answers is not in the document]
Comment (mreboud) [reply eos/b-decays/b-to-gamma-l-nu.hh:51]: I followed the notations of `"B->pilnu::d^2Gamma/dq2/dcos(theta_l)"`. We can make all of these notations consistent later.

**Pull request merged**